src/lib.rs

```diff
@@ -91,10 +91,14 @@ pub struct Assert {
     cmd: Vec<String>,
     expect_success: bool,
     expect_exit_code: Option<i32>,
-    expect_output: Option<String>,
-    fuzzy_output: bool,
-    expect_error_output: Option<String>,
-    fuzzy_error_output: bool,
+    expect_stdout: Option<OutputAssertion>,
+    expect_stderr: Option<OutputAssertion>,
+}
+
+#[derive(Debug)]
+struct OutputAssertion {
+    expect: String,
+    fuzzy: bool,
```

Comment:
I suggest we start a separate discussion about future refactoring, e.g. in the ticket #20 you created, or yet another general architecture discussion ticket?